lib/src/core/interface/signature_body_signer.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:requests_signature_dart/src/core/implementation/signature_body_parameters.dart';
+
+abstract class ISignatureBodySigner {
```

Comment:
All abstract classes must be documented.
Also, I'm noticing that you re-wrote all the documentation instead of taking the existing one from the .NET code and I don't understand why. Wouldn't it have been faster to just adapt the existing documentation from the C# code to the Dart format instead of re-writing new documentation?